DoctorWho/DoctorWho.Db/Repositories/CallsRepository.cs

```diff
@@ -0,0 +1,34 @@
+﻿
+namespace DoctorWho.Db.Repositories
+{
+    public class CallsRepository
+    {
+        private DoctorWhoCoreDbContext _context;
+        public CallsRepository(DoctorWhoCoreDbContext context)
+        {
+            _context = context;
+        }
+        public void ViewEpisodes()
```

Comment:
The method should return the episodes so the caller of this method can use them, no need for console.writeline at all in these methods :D we write these methods so we can use them from the caller services.